applications/circlemaker/src/cm_radius.erl

```diff
@@ -0,0 +1,42 @@
+%%%-------------------------------------------------------------------
+%%% @copyright (C) 2010-2015, 2600Hz
+%%% @doc
+%%%  The module generalizes an access to the eradius app.
+%%% @end
+%%% @contributors
+%%%   SIPLABS, LLC (Vladimir Potapev)
+%%%-------------------------------------------------------------------
+-module(cm_radius).
+
+%% API
+-export([add_request_authz/2, add_request_start/2, add_request_stop/2]).
+
+%%--------------------------------------------------------------------
+%% @public
+%% @doc
+%%
+%% @end
+%%--------------------------------------------------------------------
+-spec add_request_authz(wh_json:object(), boolean()) -> any().
+add_request_authz(_JObj, 'true') ->
```

Comment:
Yes, you are right, currently the Circlemaker app does nothing. We are decided to send this part of already written code to reduce size of code for the future review of the [KAZOO-3521](https://2600hz.atlassian.net/browse/KAZOO-3521).